RUN_DEPLOY.md

```diff
@@ -0,0 +1,240 @@
+<h1>Document run and deploy app react native</h1>
+
+**This guide builds for Mac OS, does not support Window OS**
```

Comment:
**NOTE**
- **This guide builds for Mac OS, haven't supported Window OS yet.**
- **The running and deploying instructions below are written based on the PIT PORT project, not the sample project.**
- **Only for references.**